src/openforms/registrations/contrib/objects_api/tests/test_api_endpoints.py

```diff
@@ -0,0 +1,146 @@
+from pathlib import Path
+from unittest.mock import patch
+
+from rest_framework import status
+from rest_framework.reverse import reverse_lazy
+from rest_framework.test import APITestCase
+
+from openforms.accounts.tests.factories import StaffUserFactory, UserFactory
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from .test_objecttypes_client import get_test_config
+
+
+class ObjecttypesAPIEndpointsTests(OFVCRMixin, APITestCase):
+
+    VCR_TEST_FILES = Path(__file__).parent / "files"
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.endpoint = reverse_lazy("api:objects_api:object-types")
```

Comment:
FYI - you can just use `reverse_lazy` with a class attribute for these kind of situations